mteb/tasks/PairClassification/deu/FalseFriendsDeEnPC.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+
+
+class FalseFriendsDeEnPC(AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="FalseFriendsGermanEnglish",
+        description="False Friends / false cognates between English and German.",
+        reference="https://drive.google.com/file/d/1jgq0nBnV-UiYNxbKNrrr2gxDEHm-DMKH/view?usp=share_link",
+        dataset={
+            "path": "aari1995/false_friends_de_en_mteb",
+            "revision": "15d6c030d3336cbb09de97b2cefc46db93262d40",
+        },
+        type="PairClassification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["deu-Latn"],
+        main_score="ap",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license="mit",
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
```

Comment:
again an example here would be nice. The paper is though already linked above.
For me it would be fine to also leave it but however you like.
author = Aaron Chibb
title = German-English False Friends in Multilingual Transformer Models- An Evaluation on Robustness and Word-to-Word Fine-Tuning
year = 2023

Reply:
There is two examples in the documentation [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md). Let me know if the documentation is lacking!

Reply:
I just saw the date is in the second example thank you! (maybe also add it to the first one). For the bibtex it is in both cases ... # removed for brevity so was not sure and still am not.

Reply:
Ahh we should probably add that in then - will do that! (#351)

Reply:
ok thanks added!

Reply:
pr should be fine now feel free to merge, I will be only availible again at thursday :)